docker/Dockerfile

```diff
@@ -116,7 +116,7 @@ RUN set -x \
   && mkdir -p $SENTRY_CONF
 
 COPY /dist/*.whl /tmp/dist/
-RUN pip install /tmp/dist/*.whl && pip check && rm -rf /tmp/dist
+RUN pip install /tmp/dist/*.whl --use-feature=2020-resolver && pip check && rm -rf /tmp/dist
```

Comment:
Hmm, we don't upgrade pip in this dockerfile, it'll probably fail because this was introduced very recently.

Reply:
Yeah: `no such option: --use-feature`

So I'd upgrade, pinned with a lower bound that has this cli kwarg. Also while you're at it, could we move the "sane defaults for pip" env vars to a separate ENV just before pip upgrade?

Reply:
>could we move the "sane defaults for pip" env vars to a separate ENV just before pip upgrade?

We could but that would create another layer, which I don't want. Do you need this for something else?

>Yeah: no such option: --use-feature

Interesting, I expected these images to be recent enough. I guess not. I'll upgrade and see how it goes from there.